components/editor/modules/list/ui.js

```diff
@@ -1,6 +1,50 @@
 import React from 'react'
-import { matchBlock, createBlockButton, buttonStyles } from '../../utils'
+import { Label, Checkbox } from '@project-r/styleguide'
+import { matchBlock, createBlockButton, buttonStyles, createPropertyForm } from '../../utils'
+import createOnFieldChange from '../../utils/createOnFieldChange'
 import injectBlock from '../../utils/injectBlock'
+import UIForm from '../../UIForm'
+
+export const ListForm = options => {
+  const Form = ({ disabled, value, onChange, t }) => {
+    if (disabled) {
+      return null
+    }
+    const list = value.blocks.reduce(
+    (memo, node) =>
+        memo || value.document.getFurthest(node.key, matchBlock(options.TYPE)),
```

Comment:
why not just `.find` if you only select one? But also what if multiple lists are in the selection?

Reply:
`find` would return the child `listItem` instead of the list itself.
Multiple lists would involve more iteration work and given the fact, that this would happen on every select, I'd avoid it until necessary.